status/permissions.nim

```diff
@@ -1,16 +1,9 @@
 import json
-import strutils
 import sets
-import statusgo_backend/core
 import chronicles
+import ./statusgo_backend/permissions as status_permissions
```

Comment:
so given this PR already cleans a lot of stuff up, this is ok for now, but later we need to review and not move this to ./backends/statusgo_backend because `statusgo_backend` should be a thin wrapper and not have too much logic which should be here instead. And what constitutes 'logic' or not in this case, which should become clearer with the mock api